ch3/recursion.js

```diff
@@ -0,0 +1,32 @@
+const assert =  require('assert');
+/**
+ * checks if input value is even
+ * @param {*} n - input value
+ * @returns true if even, false if not
+ */
+const isEven = (n) => {
+  n = Math.abs(n);
+
+  //these are so short. Can i just keep them one line? Would anyone really care?
+  if (n === 0) return true; 
+  else if (n === 1) return false;
+  return isEven(n-2);
```

Comment:
```suggestion
  return isEven(n - 2);
```

always add 1 space on both sides of operators.